checker/src/main/java/org/checkerframework/checker/mustcall/JavaEE.astub

```diff
@@ -13,6 +13,6 @@ package javax.servlet;
 interface ServletResponse {
     // The link below justifies why these annotations are correct
     // https://stackoverflow.com/questions/1159168/should-one-call-close-on-httpservletresponse-getoutputstream-getwriter
-    @NotOwning ServletOutputStream getOutputStream() throws IOException;
-    @NotOwning PrintWriter getWriter() throws IOException;
+    @MustCall({}) ServletOutputStream getOutputStream() throws IOException;
```

Comment:
Can we add comments here saying that we should switch back to `@NotOwning` once #5472 is fixed?  `@MustCall({})` works for the checker but it doesn't accurately capture what's going on here IMO.